content/news/2020-02-04.md

```diff
@@ -0,0 +1,14 @@
+---
+path: '/01-27-2020/spring-cloud-dataflow-2.4.0-released'
+title: 'Spring Cloud Data Flow 2.4.0 Released'
+date: '02-04-2020'
+category: 'release'
+image: './thumbs/default.png'
+description: 'The release 2.4.0 mainly focused on the UI updates and selected enhacements.'
+---
+
+The release 2.4.0 mainly focused on the UI updates and selected enhacements. Our post for [2.4.0.M1](/news/01-27-2020/spring-cloud-dataflow-2.4.0.M1-released) contains more info about the changes.
```

Comment:
Can we not reuse to include the blog as-is as opposed to duplicating the text? We will end up updating the blogs at times, and things might go out of sync.

Reply:
It'd be always better to keep it a single source of truth, and that truth is the actual Spring blog itself.